File: Baekjoon/Java/1.java
import java.io.*;
import java.util.*;

public class Main {
    public static void main(String[] args) throws Exception {
        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
        String s;
        StringBuilder answer = new StringBuilder();
        while ((s=br.readLine()) != null) {
            int n = Integer.parseInt(s);
            int base = 1;
            int cnt = 1;
            while ((base=base%n) != 0) {
                cnt++;
                base = base*10+1;
            }
            answer.append(cnt).append('\n');
        }
        System.out.print(answer);
    }
}

File: Baekjoon/Python/1.py
import sys
input = sys.stdin.readline

while True:
    try:
        n = int(input())
    except:
        break
    str = '1'
    cnt = 1
    while True:
        if int(str) % n == 0:
            break
        str += '1'
        cnt += 1

    print(cnt)
